scripts/import_cldr.py

```diff
@@ -16,6 +16,9 @@
 import os
 import re
 import sys
+
+from babel.dates import split_interval_pattern
```

Comment:
This must go below the `sys.path` manipulation, i.e., together with the other `babel` imputs.  Otherwise, you may import a **different** Babel version.